activities/activity_johnpolo.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+
+def run_topo_par(scanned_elev, morphmap, select_meth, env, **kwargs):
+    gs.run_command(
+        "r.param.scale",
+        input=scanned_elev,
+        output=morphmap,
+        method=select_meth,
+        env=env,
+        overwrite=True,
```

Comment:
The `overwrite=True,` is actually not needed here because you set that globally with `env["GRASS_OVERWRITE"] = "1"`.

Reply:
I deleted `"overwrite=True"`.